### PR TITLE
documentation prep for the 0.19 release

### PR DESCRIPTION
**Pull request by mlubin**
This can be merged before the release. It basically says that we're almost ready.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -9,15 +9,13 @@
...
-    a [major transition](https://discourse.julialang.org/t/mathoptinterface-and-upcoming-breaking-changes-in-jump-0-19)
-    to MathOptInterface. The development version is beta quality and the
-    documentation has TODOs remaining before the release. Please provide
-    feedback and file issues if you use this branch.
-
-    The [NEWS.md](https://github.com/JuliaOpt/JuMP.jl/blob/master/NEWS.md) file
-    contains a comprehensive list of the changes between JuMP 0.18 and JuMP 0.19
-    to aid users updating their existing models.
+    Between versions 0.18 and 0.19, JuMP underwent a major transition in its
+    underlying solver abstraction API (from MathProgBase to MathOptInterface).
```
Comment (blegat) [inline docs/src/index.md]: Maybe add hyperlinks to MPB and MOI repos

File: NEWS.md [hunk 2]
```diff
@@ -6,13 +6,8 @@ Version 0.19 (unreleased)
...
 **JuMP 0.19 contains significant breaking changes.**
 
-We will tag alpha and beta versions of 0.19 as appropriate to make it easier to
-test JuMP 0.19 before the final release. These are preview releases for early
-adopters. Additional breaking changes may occur between development tags.
-The documentation has known gaps. When using a development tag, be prepared to
-peek into the JuMP source code and tests for examples of how things work.
-
-Latest development tag: `v0.19-beta2` (`] add JuMP#v0.19-beta2`).
+We invite earlier adopters to try out the `v0.19-beta2` release
```
Comment (blegat) [inline NEWS.md]: Why keep that ? It will be in the README of JuMP v0.19.0 installations

Comment (blegat) [reply NEWS.md]: However, since the README is hidden in `.julia/packages/???/JuMP/README.md` it's might be ok to keep it and remove it after the tag

Comment (mlubin) [reply NEWS.md]: This sentence should be removed before the final tagging.

File: docs/src/index.md (hunk 1 above)
Comment (mlubin) [reply docs/src/index.md]: Done

**Pull request merged**